Cargo.toml

```diff
@@ -28,8 +28,9 @@ opt-level = 3
 
 [dev-dependencies]
 stdsimd-test = { version = "0.*", path = "stdsimd-test" }
-cupid = "0.3"
+cupid = "0.4.0"
 
 [features]
 strict = []
-std = []
\ No newline at end of file
+std = []
+intel_sde = []
```

Comment:
When you get a chance, could you add documentation here for what `intel_sde` is?

Reply:
Sure, I'll add a comment to the Cargo.toml.